uavcan_data_types/cvra/actuator/20150.Feedback.uavcan

```diff
@@ -2,6 +2,6 @@
 # Feedback coming from actuator-firmware nodes
 #
 
-float16[2] pressure       # [Pascal]
+uint33[2]  pressure       # [Pascal]
```

Comment:
A particular reason why uint33 or is this a typo?
In the code we're writing a uint32_t anyway.

Reply:
Definitely a typo.